dashboard/app/controllers/featured_projects_controller.rb

```diff
@@ -14,4 +15,40 @@ def unfeature
     @featured_project = FeaturedProject.find_by! storage_app_id: storage_app_id
     @featured_project.update! unfeatured_at: DateTime.now
   end
+
+  # Featured projects are selected internally for their
+  # quality, so we can be reasonably confident that they
+  # are not abusive. To prevent users from spamming Zendesk # with false reports of abuse on featured projects, this # sets their abuse score such that the project needs to
+  # be reported many times before being blocked.
+  def buffer_abuse_score(score=-50)
+    channels_path = "/v3/channels/#{params[:project_id]}/buffer_abuse_score"
+    assets_path = "/v3/assets/#{params[:project_id]}/"
+    files_path = "/v3/files/#{params[:project_id]}/"
+
+    ChannelsApi.call(
+      'REQUEST_METHOD' => 'POST',
+      'PATH_INFO' => channels_path,
+      'REQUEST_PATH' => channels_path,
+      'HTTP_COOKIE' => request.env['HTTP_COOKIE'],
+      'rack.input' => StringIO.new
+    )
+
+    FilesApi.call(
```

Comment:
why do we need to make these 2 calls to the FilesApi here?

Reply:
great question - I included them because they were in https://github.com/code-dot-org/code-dot-org/blob/a052b1cc1a97555f1db6a039b6e2c1fb808c6f59/dashboard/app/controllers/report_abuse_controller.rb#L69 but that was maybe a bad assumption. I'll slow down and see what I can figure out if/why they are needed.

Reply:
also if all 3 calls are needed in both places, I should probably find a way to re-use that code?

Reply:
I haven't looked closely, but generally speaking when we mark a project as abusive we may also mark its assets/files as abusive so that direct links to those images no longer work. so the scenario to test here is an applab, weblab or gamelab project with images / animations gets featured and then reported for abuse twice. do the images / animations still appear correctly in that project after report abuse is clicked twice? do direct urls to images still stop working after the project is reported for abuse 7 times?

Reply:
Did some sleuthing and agree with Dave's conclusion. Also, did some manual testing that this works as expected for images/animations based on the suggestions. But in doing so, also realized that when we fetch projects for the gallery we only fetch projects with an abuse score of 0 - with this change, we'll need projects with abuse scores of zero _or less_.  Working on making that fix now.